Cartfile

```diff
@@ -1 +1 @@
-github "getsentry/sentry-cocoa" "6.0.10"
+github "getsentry/sentry-cocoa" "8.16.1"
```

Comment:
I had to update Sentry due to an error:

```
Projects/BlockBlock/Carthage/Checkouts/sentry-cocoa/Sources/Sentry/SentrySwizzle.m:104:26: error: a function declaration without a prototype is deprecated in all versions of C [-Werror,-Wstrict-prototypes]
swizzledClassesDictionary()
                         ^
                          void
```

Reply:
I'm going to remove Sentry as 
1. I don't like the external dependency (that as you pointed out, often needs updating)
2. It raises GDPR concerns

Reply:
Thank you for your answer. I can remove it easily in a separate MR, along with Carthage integration 👍🏻

Reply:
Reverted the change